telemetry-aware-scheduling/pkg/telemetryscheduler/telemetryscheduler.go

```diff
@@ -193,8 +193,10 @@ func (m MetricsExtender) filterNodes(args extender.Args) *extender.FilterResult
 	}
 	dontscheduleStrategy, err := m.getDontScheduleStrategy(policy)
 	if err != nil {
-		klog.V(2).InfoS("Don't scheduler strategy failed "+err.Error(), "component", "extender")
-		return nil
+		klog.V(2).InfoS("Returning all nodes "+err.Error(), "component", "extender")
```

Comment:
What do you think about increasing the log level a bit higher than 2? This isn't really an error and it is pretty common not to use the dontschedule strategy. I'd rather prefer it printing only when someone turns on all the logs. Seems 3 is not used so maybe 4?